src/components/selection-list/basic-list.tsx

```diff
@@ -0,0 +1,112 @@
+import React = require('react');
+import ReactDOM  = require('react-dom');
+import {SBComponent} from 'stylable-react-component';
+import {root} from 'wix-react-tools';
+import {noop} from '../../utils';
+import listStyle from './selection-list.st.css';
+
+export type ItemValue = string;
+
+function closestElementMatching(
+    predicate: (element: HTMLElement) => boolean,
+    startAt: HTMLElement
+): HTMLElement | null {
+    let current: HTMLElement | null = startAt;
+    while (current && !predicate(current)) {
+        current = current.parentElement;
+    }
+    return current;
+}
+
+export function getFocusableItemValues(nodes: React.ReactNode): ItemValue[] {
+    const result: ItemValue[] = [];
+    React.Children.forEach(nodes, node => {
+        if (node && typeof node === 'object') {
+            const element = node as React.ReactElement<any>;
```

Comment:
Without casting to ReactElement TypeScript would complain that property 'props' doesn't exist on type.